src/init.cpp

```diff
@@ -803,6 +805,10 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
             return InitError(strprintf(_("Invalid amount for -minrelaytxfee=<amount>: '%s'"), mapArgs["-minrelaytxfee"]));
     }
 
+    fRequireStandard = !GetBoolArg("-acceptnonstdtxn", !Params().RequireStandard());
+    if (Params().RequireStandard() && !fRequireStandard)
```

Comment:
Anyway, enough discussion please ack/nack https://github.com/jtimon/bitcoin/commit/2d71478a68d3a2c6cb5f5210f76b37d1f60a22cf in #6068 so that we can move on.